src/backend/resourcemanager/communication/rmcomm_QD2RM.c

```diff
@@ -2145,7 +2145,7 @@ int findFile(const char *filename)
 int createFile(const char *filename)
 {
 	int fd = 0;
-	fd = open(filename, O_RDWR|O_CREAT);
+	fd = open(filename, O_RDWR|O_CREAT, S_IRUSR | S_IWUSR);
```

Comment:
ok which file is it?  Is it safe to assume read and write permissions is what we want?